.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I cannot apply code style rules from the configuration file because several eslint plugin packages are absent